sigoptlite/models.py

```diff
@@ -71,10 +71,10 @@ class LocalTask:
 @dataclass(frozen=True, kw_only=True)
 class LocalParameterPrior:
   name: str
-  mean: float = None
-  scale: float = None
-  shape_a: float = None
-  shape_b: float = None
+  mean: float | None = None
```

Comment:
I see, so the type for `mean` is float or None.

Reply:
correct. If you think `None` should not be allowed I can remove `| None = None`

Reply:
I don't really have a preference. cc @malkomes, what do you think here?

Reply:
the `or` is fine by me.